body.adoc

```diff
@@ -77,12 +77,33 @@ include::adoc_event_tables/tlb_spec_metrics.adoc[]
 
 This group contains events and metrics related for Topdown Microarchitecture Analysis (TMA) methodology.
 
-The TMA methodology categorizes CPU execution time at a high level first. This step flags (reports high fraction value) some domain(s) for possible investigation. Next, the user can drill down into those flagged domains, and can safely ignore all non-flagged domains. The process is repeated in a hierarchical manner until a specific performance issue is determined or at least a small subset of candidate issues is identified for potential investigation.
+TMA allows even developers with minimal microarchitecture knowledge to understand, for a given workload, where bottlenecks reside.  It does so by accounting for the utilization of each pipeline "slot" in the microarchitecture.  When the hardware is utilized with optimal efficiency, each slot is occupied by an instruction or micro-operation (uop) that will go on to execute and retire.  When bottlenecks occur, due perhaps to a cache miss, branch misprediction, or any number of other microarchitectural conditions, some slots may be either unused or thrown away, which results in inefficiency and reduced performance.  TMA is able to identify these wasted slots, and the stalls, clears, misses, or other events that cause them.  This enables developers to make informed decisions when tuning their code.
```

Comment:
To my experience people may have problems understanding "slot" concept. What about adding sentence like following:
"The number slots on each cycle is equal to pipeline width. E.g. a 4-wide machine will have 4 slots each cycle." 

We can also link original article from Ahmad about Top Down analysis (https://ieeexplore.ieee.org/document/6844459).

Reply:
See my latest commit, adds such a clarification, as well as links to various top-down papers.